lib/pronto/config.rb

```diff
@@ -40,7 +40,7 @@ def bitbucket_hostname
     end
 
     def max_warnings
-      ENV['PRONTO_MAX_WARNINGS'] || @config_hash['max_warnings']
+      ENV['PRONTO_MAX_WARNINGS'] && ENV['PRONTO_MAX_WARNINGS'].to_i || @config_hash['max_warnings']
```

Comment:
Consider
```ruby
ENV['PRONTO_MAX_WARNINGS'] && Integer(ENV['PRONTO_MAX_WARNINGS']) || @config_hash['max_warnings']
```
so only integer values in `ENV['PRONTO_MAX_WARNINGS']` will be allowed.

Reply:
I like it, updating the PR.